slang-antlr/src/test/java/com/sonarsource/slang/antlr/SLangConverterTest.java

```diff
@@ -67,7 +70,7 @@ public void testConverter() throws IOException {
 
     assertTree(tree).isInstanceOf(TopLevelTree.class).hasChildren(9);
     assertThat(numBinNodes.get()).isEqualTo(6);
-    assertThat(numIdentifierNode.get()).isEqualTo(18);
+    assertThat(numIdentifierNode.get()).isEqualTo(19);
```

Comment:
We should do something about this test case, maybe just drop it.

Reply:
Ok, will drop it. It does not impact coverage on classes in `slang-antlr` module